package/freescale-imx/imx-seco/Config.in

```diff
@@ -0,0 +1,9 @@
+config BR2_PACKAGE_IMX_SECO
```

Comment:
This was put in the external packages area in emb-platform since this hasn't been upstreamed - likely might be better to just update that version until an upstream package shows up.

Reply:
> This was put in the external packages area in emb-platform since this hasn't been upstreamed - likely might be better to just update that version until an upstream package shows up.
> 
> Check emb-platform/os/buildroot/packages/imx-seco

imx-seco is upstreamed into buildroot here and is reason to remove from emb-platform. The only thing I had to change is to use firmware for rev "C0" CPU as buildroot default is still "B0". I removed one from emb-platform as part of ref update(To be PR after this merge).

Reply:
but it is not upstreamed in upstream - by continuing to fork from the upstream - it just makes rebasing to upstream buildroot 2021.02.x for example a bit more of a challenge.

Reply:
> but it is not upstreamed in upstream - by continuing to fork from the upstream - it just makes rebasing to upstream buildroot 2021.02.x for example a bit more of a challenge.

imx-seco is up streamed into 2020.11.x, and master. So 2021.02.x rebase wouldn't be conflict.